openapi.yml

```diff
@@ -1154,13 +1154,30 @@ paths:
       operationId: admin-get-handler
       tags:
         - Admin
-      description: ""
+      description: Gets a handler
     parameters:
       - schema:
           type: string
         name: id
         in: path
         required: true
+    delete:
+      summary: ""
+      operationId: admin-delete-handler
+      responses:
+        "200":
+          description: OK
+          content:
```

Comment:
Don't return a body for this API

Reply:
@JoshuaWilkes Done, and looked into the correct response to use for delete (with no content) and most say to use 204 response. So have updated to use that.